### PR TITLE
Giddh multilanguage

### PR DESCRIPTION
**Pull request by ravinderwalkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/audit-logs/components/audit-logs-table/audit-logs-table.component.ts [hunk 1]
```diff
@@ -1,19 +1,23 @@
-import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Component, OnDestroy, OnInit, Input } from '@angular/core';
```
[the remark this answers is not in the document]
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/audit-logs/components/audit-logs-table/audit-logs-table.component.ts:1]: Done

File: apps/web-giddh/src/app/import-excel/upload-success/upload-success.component.ts [hunk 2]
```diff
@@ -41,17 +48,25 @@ export class UploadSuccessComponent implements OnInit, OnDestroy {
 					}
 				} else {
 					this.selectedType = 'accounts';
-				}
+                }
+                
+                this.importedCountMessage = this.localeData.imported_count_message;
```
[the remark this answers is not in the document]
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/import-excel/upload-success/upload-success.component.ts:53]: Done

File: apps/web-giddh/src/app/import-excel/upload-success/upload-success.component.ts [hunk 3]
```diff
@@ -41,17 +48,25 @@ export class UploadSuccessComponent implements OnInit, OnDestroy {
...
+                this.importedCountMessage = this.importedCountMessage.replace("[SUCCESS_COUNT]", String(this.UploadExceltableResponse.successCount));
+                this.importedCountMessage = this.importedCountMessage.replace("[COUNT]", String(this.UploadExceltableResponse.failureCount + this.UploadExceltableResponse.successCount));
+                this.importedCountMessage = this.importedCountMessage.replace("[SELECTED_TYPE]", this.selectedType);
+                this.importedCountMessage = this.importedCountMessage.replace("[IS_ARE]", this.isAre);
+
+                this.failedReportMessage = this.localeData.failed_report_message;
```
[the remark this answers is not in the document]
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/import-excel/upload-success/upload-success.component.ts:59]: Done

File: apps/web-giddh/src/app/new-vs-old-Invoices/new-vs-old-Invoices.component.ts [hunk 4]
```diff
@@ -47,6 +47,11 @@ export class NewVsOldInvoicesComponent implements OnInit, OnDestroy {
     @ViewChild('paginationChild', {static: true}) public paginationChild: ElementViewContainerRef;
     /* Observable to unsubscribe all the store listeners to avoid memory leaks */
     private destroyed$: ReplaySubject<boolean> = new ReplaySubject(1);
+    /* This will hold local JSON data */
+    public localeData: any = {};
+    /* This will hold common JSON data */
+    public commonLocaleData: any = {};
+    public bifurcationClients: string = "";
```
[the remark this answers is not in the document]
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/new-vs-old-Invoices/new-vs-old-Invoices.component.ts:54]: Done

File: apps/web-giddh/src/app/permissions/components/confirmation/confirmation.model.component.ts [hunk 5]
```diff
@@ -1,16 +1,32 @@
...
+export class DeleteRoleConfirmationModelComponent implements OnInit {
     @Input() public selectedRoleForDelete: IRoleCommonResponseAndRequest;
+    /* This will hold local JSON data */
+    @Input() public localeData: any = {};
+    /* This will hold common JSON data */
+    @Input() public commonLocaleData: any = {};
     @Output() public confirmDeleteEvent: EventEmitter<boolean> = new EventEmitter(true);
     @Output() public closeModelEvent: EventEmitter<boolean> = new EventEmitter(true);
 
+    public confirmationMessage: string = "";
```
[the remark this answers is not in the document]
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/permissions/components/confirmation/confirmation.model.component.ts:18]: Done

File: apps/web-giddh/src/app/permissions/components/confirmation/confirmation.model.component.ts [hunk 6]
```diff
@@ -1,16 +1,32 @@
...
+
+    public ngOnInit(): void {
```
[the remark this answers is not in the document]
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/permissions/components/confirmation/confirmation.model.component.ts:20]: Done

File: apps/web-giddh/src/app/shared/header/components/account-operations/account-operations.component.ts [hunk 7]
```diff
@@ -38,7 +38,14 @@ import { ShSelectComponent } from '../../../../theme/ng-virtual-select/sh-select
     templateUrl: './account-operations.component.html',
     styleUrls: [`./account-operations.component.scss`]
 })
+
 export class AccountOperationsComponent implements OnInit, AfterViewInit, OnDestroy, OnChanges {
+    /* This will hold local JSON data */
+    @Input() public localeData: any = {};
+    /* This will hold common JSON data */
+    @Input() public commonLocaleData: any = {};
+    public groupSharedWith: string = "";
```
[the remark this answers is not in the document]
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/shared/header/components/account-operations/account-operations.component.ts:47]: Done

File: apps/web-giddh/src/app/shared/header/components/account-operations/account-operations.component.ts [hunk 8]
```diff
@@ -393,6 +400,20 @@ export class AccountOperationsComponent implements OnInit, AfterViewInit, OnDest
         this.accountsAction.mergeAccountResponse$.pipe(takeUntil(this.destroyed$)).subscribe(res => {
             this.selectedaccountForMerge = '';
         });
+
+        this.activeGroupSharedWith$.subscribe(response => {
+            if(response) {
+                this.groupSharedWith = this.localeData.shared_with;
```
[the remark this answers is not in the document]
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/shared/header/components/account-operations/account-operations.component.ts:406]: Done

File: apps/web-giddh/src/app/shared/header/components/account-operations/account-operations.component.ts [hunk 9]
```diff
@@ -393,6 +400,20 @@ export class AccountOperationsComponent implements OnInit, AfterViewInit, OnDest
...
+                this.groupSharedWith = this.groupSharedWith.replace("[ACCOUNT_GROUPS_COUNT]", String(response.length));
+            }
+        });
+
+        this.activeAccountSharedWith$.subscribe(response => {
+            if(response) {
+                this.accountSharedWith = this.localeData.shared_with;
```
[the remark this answers is not in the document]
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/shared/header/components/account-operations/account-operations.component.ts:413]: Done

File: apps/web-giddh/src/app/shared/header/components/account-operations/account-operations.component.ts [hunk 10]
```diff
@@ -677,7 +698,8 @@ export class AccountOperationsComponent implements OnInit, AfterViewInit, OnDest
 
     public showDeleteMergedAccountModal(merge: string) {
         merge = merge.trim();
-        this.deleteMergedAccountModalBody = `Are you sure you want to delete ${merge} Account ?`;
+        this.deleteMergedAccountModalBody = this.localeData.delete_merged_account_content;
```
[the remark this answers is not in the document]
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/shared/header/components/account-operations/account-operations.component.ts:701]: Done

File: apps/web-giddh/src/app/shared/header/components/account-operations/account-operations.component.ts [hunk 11]
```diff
@@ -728,13 +750,15 @@ export class AccountOperationsComponent implements OnInit, AfterViewInit, OnDest
...
         } else {
-            this._toaster.errorToast('Please Select at least one account');
+            this._toaster.errorToast(this.localeData.merge_account_error);
             return;
         }
     }
 
     public showMoveMergedAccountModal() {
-        this.moveMergedAccountModalBody = `Are you sure you want to move ${this.setAccountForMove} Account to ${this.selectedAccountForMove} ?`;
+        this.moveMergedAccountModalBody = this.localeData.move_merged_account_content;
```
[the remark this answers is not in the document]
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/shared/header/components/account-operations/account-operations.component.ts:759]: Done

File: apps/web-giddh/src/app/userDetails/components/subscriptions/subscriptions.component.ts [hunk 12]
```diff
@@ -30,6 +30,8 @@ export class SubscriptionsComponent implements OnInit, OnChanges, AfterViewInit,
 
     /* This will have active tab value */
     @Input() public activeTab: string = '';
+    @Input() public localeData: any = {};
```
[the remark this answers is not in the document]
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/userDetails/components/subscriptions/subscriptions.component.ts:33]: Done

**Review by prateek-giddh — APPROVED** [review]

**Pull request merged**